hmpps-prisoner-search/helm_deploy/values-prod.yaml

```diff
@@ -1,14 +1,14 @@
----
 generic-service:
   ingress:
     host: prisoner-search.prison.service.justice.gov.uk
 
   # Additional allowlist rules
   # See also allowlist values helm_deploy/prisoner-offender-search/values.yaml
   allowlist:
-    prum-prod-platform-1: "18.170.20.75/32"
-    prum-prod-platform-2: "35.177.223.199/32"
-    prum-prod-platform-3: "18.170.155.82/32"
+    prum-prod-platform-1: 18.170.20.75/32
+    prum-prod-platform-2: 35.177.223.199/32
+    prum-prod-platform-3: 18.170.155.82/32
+    groups: []
```

Comment:
@andrewrlee this doesn't look right - overwriting the `groups: internal` from the `values.yml` with an empty array?

Reply:
I've fixed these if you can re-approve @petergphillips